### PR TITLE
Code was updated for the сheckChangesSchemaEndpoint feature file

### PR DESCRIPTION
**Pull request by mpushnikA1QA**
Code was updated for the сheckChangesSchemaEndpoint feature file

### CODE REVIEW

File: src/test/features/сheckCountSchemaEndpoint.feature [hunk 1]
```diff
@@ -1,6 +1,5 @@
-@automated @PBPA-1780 @PBPA-1781 @prod
-Feature: Count schema endpoint for the feed and search common endpoints for the API entity
-
+Feature: /search and feed/[entity]/count/schema endpoint for the API entities
+@automated @PBPA-1780 @prod
```
Comment (danil-shamsiev) [inline src/test/features/сheckCountSchemaEndpoint.feature:2]: Put an empty line between Feature heading and tags-line. Align tags with Scenario Outline

File: src/test/features/сheckCountSchemaEndpoint.feature [hunk 2]
```diff
@@ -17,6 +16,7 @@ Feature: Count schema endpoint for the feed and search common endpoints for the
             | drugTrends            |
             | organizationHierarchy |
 
+@automated @PBPA-1781 @prod
```
Comment (danil-shamsiev) [inline src/test/features/сheckCountSchemaEndpoint.feature:19]: Align with Scenario Outline

**Review by danil-shamsiev — CHANGES_REQUESTED** [review]

File: src/test/features/сheckCountSchemaEndpoint.feature (hunk 1 above)
Comment (mpushnikA1QA) [reply src/test/features/сheckCountSchemaEndpoint.feature]: The code was updated

File: src/test/features/сheckCountSchemaEndpoint.feature (hunk 2 above)
Comment (mpushnikA1QA) [reply src/test/features/сheckCountSchemaEndpoint.feature]: The code was updated

**Review by danil-shamsiev — APPROVED** [review]

**Pull request merged**